.github/workflows/ios-ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: macos-10.15
+
+    steps:
+    # Checkout the repo
+    - uses: actions/checkout@v2
+
+    - name: brew install
+      run: brew install clang-format
+
+    - name: npm install
+      run: |
+        npm ci
+        if [ -d ./lib/node_modules ]; then cp -rf ./lib/node_modules/* ./node_modules/; fi
```

Comment:
Why are we doing this? If npm is putting node_modules there I don't think we should be messing with it. It doesn't seem to cause any problems.

Reply:
It does, because we assume the location of the font files

Reply:
Can you try to rebase? @k0nserv deduped our dependencies to fix that.

Reply:
Actually the PR is not merged yet https://github.com/Skyscanner/backpack-react-native/pull/451

Reply:
Yeah I don't think this should be the solution to this issue. The problem is that we expect the font files to be in `node_modules/bpk-svgs` but they ended up nested under `backpack-react-native` at `node_modules/backpack-react-native/node_modules/bpk-svgs`. I ran `npm dedupe bpk-svgs` in my branch which moves them to the root

Reply:
I just merged #451 so if you rebase on master that should be it